src/server/resolve-options.js

```diff
@@ -0,0 +1,39 @@
+export function resolveOptions (options, done) {
```

Comment:
Unfortunately we don't actually need this. I didn't explain it clear enough, but the proper usage would be bundling the root component via Webpack with `target: 'node'`, which already automatically shims `require.ensure` and `require(['...'], cb)` into synchronous requires. So as long as you stick to Webpack's supported code splitting syntax, you don't need to do anything.